controllers/replication-controller/dellcsireplicationgroup_controller.go

```diff
@@ -142,7 +145,7 @@ func (r *ReplicationGroupReconciler) Reconcile(ctx context.Context, req ctrl.Req
 	}
 
 	// Try to get the client
-	remoteClient, err := r.Config.GetConnection(remoteClusterID)
+	client, err := r.Config.GetConnection(remoteClusterID)
```

Comment:
**suggestion:** Can we revert this back to `remoteClient` since the same Reconcile is done for single and multi-cluster?